gcp/appengine/deploy.sh

```diff
@@ -21,17 +24,14 @@ for config in $configs_dir/*.yaml; do
 done
 
 cd "$dir"
+image="gcr.io/$project_id/osv-website"
+
+if [ -z "$CLOUDBUILD" ]; then
+  pushd ../../
+  # Using BuildKit allows us to cache the multi-stage builds
+  DOCKER_BUILDKIT=1 docker build --build-arg=BUILDKIT_INLINE_CACHE=1 -t $image:$version -f gcp/appengine/Dockerfile --cache-from=$image .
```

Comment:
```suggestion
  DOCKER_BUILDKIT=1 docker build --build-arg=BUILDKIT_INLINE_CACHE=1 -t "${image}:${version}" -f gcp/appengine/Dockerfile --cache-from="${image}" .
```
Comply with:
- https://google.github.io/styleguide/shellguide.html#variable-expansion
- https://google.github.io/styleguide/shellguide.html#quoting